src/System.IO.FileSystem/tests/Performance/Perf.Directory.cs

```diff
@@ -58,33 +58,50 @@ public void Exists()
             Directory.Delete(testFile);
         }
 
-        [Benchmark]
-        [InlineData(10, 100)]
-        [InlineData(100, 10)]
-        [InlineData(1000, 1)]
-        [OuterLoop("Takes a lot of time to finish")]
-        public void RecursiveCreateDirectory(int depth, int times)
+        public string GetTestDeepFilePath(int depth)
```

Comment:
It would be better to pass in the base path and build the full string- you're just throwing away the string here otherwise

Reply:
Never mind, I see that you're modifying it on each loop